presidio-analyzer/analyzer/analysis_explanation.py

```diff
@@ -0,0 +1,38 @@
+class AnalysisExplanation:
+    """ AnalysisExplanation is a class that holds forensic information that
```

Comment:
Not sure 'forensic' is the right word here, Dexter :)

Reply:
🤣